orchagent/intfsorch.cpp

```diff
@@ -159,6 +160,19 @@ string IntfsOrch::getRouterIntfsAlias(const IpAddress &ip, const string &vrf_nam
     return string();
 }
 
+bool IntfsOrch::isInbandIntfInMgmtVrf(const string& alias)
+{
+    if (m_syncdIntfses.find(alias) == m_syncdIntfses.end())
+        return false;
+
+    string vrf_name = "";
+    vrf_name = m_syncdIntfses[alias].vrf_name;
+    if ((!vrf_name.empty()) && (vrf_name == MGMT_VRF))
```

Comment:
open/close brace. Same for other parts of code

Reply:
done